pkg/apis/config/types.go

```diff
@@ -143,3 +143,16 @@ type NodeResourceTopologyMatchArgs struct {
 	// Namespaces to be considered by NodeResourceTopologyMatch plugin
 	Namespaces []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// NodeResourceTopologyResourceAllocationScoreArgs holds arguments used to configure the ResourceAllocationScore plugin
+type NodeResourceTopologyResourceAllocationScoreArgs struct {
+	metav1.TypeMeta
+
+	KubeConfigPath          string
+	MasterOverride          string
+	Namespaces              []string
+	ScoreSchedulingStrategy string
+	Resources               []schedulerconfig.ResourceSpec
```

Comment:
Can you add some comments to explain the new fields? (Resources and ScoringStrategy)

Reply:
Done